contrib/kubernetes/skydive.yaml

```diff
@@ -102,6 +98,11 @@ spec:
           initialDelaySeconds: 20
           periodSeconds: 10
           failureThreshold: 10
+        env:
+        - name: SKYDIVE_ANALYZER_LISTEN
+          value: "0.0.0.0:$(SKYDIVE_ANALYZER_SERVICE_PORT_API)"
```

Comment:
Where is `SKYDIVE_ANALYZER_SERVICE_PORT_API` defined ?

Reply:
https://kubernetes.io/docs/concepts/services-networking/service/#environment-variables